### DISCUSSION TITLE
v1.10.0

### DISCUSSION THREAD

**Discussion by JohnRoesler** [go-co-op/gocron#261] [Releases]
## What's Changed
* CI check for formatting by @snOm3ad in https://github.com/go-co-op/gocron/pull/240
* add IsRunning() to report the job's state by @JohnRoesler in https://github.com/go-co-op/gocron/pull/233
* add checkout step to formatting action by @snOm3ad in https://github.com/go-co-op/gocron/pull/241
* fix some wording and formatting by @JohnRoesler in https://github.com/go-co-op/gocron/pull/247
* upgrade to go 1.17 by @JohnRoesler in https://github.com/go-co-op/gocron/pull/248
* added tests for executor and gocron by @GaruGaru in https://github.com/go-co-op/gocron/pull/249
* Append tags rather than overwriting by @calvinmclean in https://github.com/go-co-op/gocron/pull/253
* Add .MonthLastDay() to schedule a task at the end of each month by @mrngm in https://github.com/go-co-op/gocron/pull/250

## New Contributors
* @snOm3ad made their first contribution in https://github.com/go-co-op/gocron/pull/240
* @GaruGaru made their first contribution in https://github.com/go-co-op/gocron/pull/249
* @mrngm made their first contribution in https://github.com/go-co-op/gocron/pull/250

**Full Changelog**: https://github.com/go-co-op/gocron/compare/v1.9.0...v1.10.0

<hr /><em>This discussion was created from the release <a href='https://github.com/go-co-op/gocron/releases/tag/v1.10.0'>v1.10.0</a>.</em>

**Comment by okridgway** [comment]
This might be a stupid question, but will this work on windows as well as linux?

**Reply by JohnRoesler** [reply]
Yes! Like all go programs you can compile for windows. It doesn’t rely on Linux cron or anything, but rather a go library that implemented cron logic using the standard library.

**Reply by okridgway** [reply]
In that case, I seem to be having another issue, as I can't get a super basic program to work.  I'll go ahead and create an "Issue" then. Thanks!
